test/needle-api/needle-client-webpack.spec.js

```diff
@@ -50,8 +50,8 @@ const mockBlueprintSubGen = class extends ClientGenerator {
 };
 
 describe('needle API Webpack: JHipster client generator with blueprint', () => {
-    before(done => {
-        helpers
+    function generateAppWithClientFramework(clientFramework) {
+        return helpers
             .run(path.join(__dirname, '../../generators/client'))
```

Comment:
```suggestion
            .create(path.join(__dirname, '../../generators/client'))
```

Reply:
What's the difference between doing .run(..) or .create(..).run() ? :)

Reply:
`helpers.run`:
- uses root generator event listeners, so if others generators fails can create a false success or force the test to timeout.

`helpers.create`
- `run()` is a new api using Environment new promised apis.

Reply:
Oh ok, didn't know about that, thanks for the explanation